styles/helpers/mediaqueries.css

```diff
@@ -1,3 +1,19 @@
 :root {
+  @custom-media --desktop-wide-up (min-width: 1440px);
```

Comment:
How would something like that work?  All of the JS mixins I've seen us create output some sort of CSS declaration that is wrapped in the selector when you use the mixin as in the helpers.

ie. 
```
@grid

# outputs

.col-12 {
  float: left;
  width: 100%;
}
```

This needs to just output the `@custom-media --desktop-up (min-width: var(--desktop-min)` type of thing where you'd have to pass in content.  In PostCSS mixins you have the `@mixin-content` for these kinds of things but how would you do that in JS?